### ISSUE TITLE
Changes not affecting my game

### ISSUE THREAD

**Issue by Shikouway** [maruohon/tweakeroo#384]
I loaded tweakeroo via curseforge, and I can access the menu for it and change stuff in the menu, but changes I make in the tweakeroo menu seem to not affect my game. I have litematica installed as well and that one works perfectly fine. Any help is appreciated.

**Comment by maruohon** [comment]
I can't do or say anything about anything if you don't specify any information at all. You didn't even mention *what doesn't work*, i.e. which features you are referring to. Also I can only assume you are referring to the 0.14.3 version for 1.19.2?

**Comment by Shikouway** [comment]
@maruohon it is for 1.19.2 and as far as I could tell none of the options would work. I tried turning on the shulker preview and zoom which I know did not work.

**Comment by maruohon** [comment]
Did you also hold the Hotkeys -> `zoomActivate` key to actually try to use the zoom? The tweak toggle is just a main option to enable/disable that activation key, as sometimes you don't want it to work as zoom when you need to press the same key for something else. And the shulker box preview is only for the items inside your inventory, and by default you need to hold shift for it to render.

**Comment by Shikouway** [comment]
Yes I did hold the hot key for zoom and tried changing the binds to a couple different keys. And for the shucker box thing I tried holding shift and also tried turning on the preview without holding shift option and neither of those worked.

**Comment by maruohon** [comment]
Do you have lots of other mods installed? Sometimes some of the features might conflict with other mods, but these two shouldn't be that common conflicts.

Although now that I think about it, there was someone on Discord just recently who had the shulker box preview not work when they had some kind of server-side mod for custom items, which wrapped the NBT data on the items such that Tweakeroo/malilib didn't see the data tag in the same location as it would be in vanilla.

**Comment by Shikouway** [comment]
I’m just stupid, there were some other settings in the mod that I had to turn on… sorry for wasting ur time.

Sent from my iPhone

On Dec 9, 2022, at 3:35 AM, Matti Ruohonen ***@***.***> wrote:

﻿

Do you have lots of other mods installed? Sometimes some of the features might conflict with other mods, but these two shouldn't be that common conflicts.

Although now that I think about it, there was someone on Discord just recently who had the shulker box preview not work when they had some kind of server-side mod for custom items, which wrapped the NBT data on the items such that Tweakeroo/malilib didn't see the data tag in the same location as it would be in vanilla.
